cucumber/api/features/member_list.feature

```diff
@@ -0,0 +1,204 @@
+     When I successfully GET "/roles/cucumber/group/dev?members&search=non_existent_user"
+     Then the JSON should be: 
+         """
+         []
+         """      
+  
+  Scenario: Page role members
+    When I successfully GET "/roles/cucumber/group/dev?members&limit=3"
+    Then the JSON should be:
+        """
+        [
+        {
+            "admin_option": true,
+            "member": "cucumber:user:admin",
+            "ownership": true,
+            "policy": "cucumber:policy:root",
+            "role": "cucumber:group:dev"
+        },
+        {
+            "admin_option": false,
+            "member": "cucumber:user:alice",
+            "ownership": false,
+            "policy": "cucumber:policy:root",
+            "role": "cucumber:group:dev"
+        },
+        {
+            "admin_option": false,
+            "member": "cucumber:user:alicia",
+            "ownership": false,
+            "policy": "cucumber:policy:root",
+            "role": "cucumber:group:dev"
+        }
+        ]
+        """
+
+    When I successfully GET "/roles/cucumber/group/dev?members&limit=3&offset=3"
+    Then the JSON should be:
+        """
+        [
+        {
+            "admin_option": false,
+            "member": "cucumber:user:bob",
+            "ownership": false,
+            "policy": "cucumber:policy:root",
+            "role": "cucumber:group:dev"
+        },
+        {
+            "admin_option": false,
+            "member": "cucumber:user:charlotte",
+            "ownership": false,
+            "policy": "cucumber:policy:root",
+            "role": "cucumber:group:dev"
+        }         
+        ]
+        """
+
+  Scenario: Counting role members
+    When I successfully GET "/roles/cucumber/group/dev?members&count=true"
+    Then the JSON should be:
+    """
+    {
+        "count": 5 
+    }
+    """
+
+    When I successfully GET "/roles/cucumber/group/dev?members&count=true&limit=3"
+    Then the JSON should be:
+    """
+    {
+        "count": 5 
+    }
+    """
+
+  Scenario: Filter role members by kind
+    When I successfully GET "/roles/cucumber/group/employees?members&kind[]=group&kind[]=user"
+    Then the JSON should be:
+    """
+    [
+        {
+          "admin_option": true,
+          "member": "cucumber:user:admin",
+          "ownership": true,
+          "policy": "cucumber:policy:root",
+          "role": "cucumber:group:employees"
+        },
+        {
+            "admin_option": false,
+            "member": "cucumber:group:dev",
+            "ownership": false,
+            "policy": "cucumber:policy:root",
+            "role": "cucumber:group:employees"
+        },
+        {
+            "admin_option": false,
+            "member": "cucumber:user:alice",
+            "ownership": false,
+            "policy": "cucumber:policy:root",
+            "role": "cucumber:group:employees"
+        }
+    ]
+    """
+
+    When I successfully GET "/roles/cucumber/group/employees?members&kind[]=group"
```

Comment:
Will this also work with a plain `kind` parameter?

Reply:
Yes, and I added a test case for this. Thanks!